easybuild/easyconfigs/e/ELPA/ELPA-2017.11.001-foss-2018b.eb

```diff
@@ -0,0 +1,23 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Authors::   Inge Gutheil <i.gutheil@fz-juelich.de>, Alan O'Cais <a.ocais@fz-juelich.de>
+# License::   MIT/GPL
+# $Id$
+#
+##
+
+name = 'ELPA'
+version = '2017.11.001'
+
+homepage = 'http://elpa.mpcdf.mpg.de'
+description = """Eigenvalue SoLvers for Petaflop-Applications ."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+toolchainopts = {'usempi': True}
```

Comment:
@micaeljtoliveira Is this necessary now or can it be removed?

Reply:
@ocaisa At the moment it is still needed, as we use `LIBSCALAPACK` and that is only set if `'usempi': True`. Maybe there is another way of doing this?